charts/jenkins/values.yaml

```diff
@@ -218,13 +218,13 @@ controller:
 
   # List of plugins to be install during Jenkins controller start
   installPlugins:
-    - kubernetes:1.29.4
+    - kubernetes:1.30.1
     - workflow-aggregator:2.6
-    - git:4.7.1
-    - configuration-as-code:1.51
+    - git:4.8.2
+    - configuration-as-code:1.52
 
   # Set to false to download the minimum required version of all dependencies.
-  installLatestPlugins: false
+  installLatestPlugins: true
```

Comment:
true is also the default of the `jenkins-plugin-cli`
>  --latest VALUE                              : Set to true to download the latest version of all dependencies, even if the version(s) of the
>                                                requested plugin(s) are not the latest. By default, plugin dependency versions will be determined by
>                                                the update center metadata or plugin MANIFEST.MF (default: true)
>

Reply:
it was changed to false because of a bug in pmcli, https://github.com/jenkinsci/plugin-installation-manager-tool/issues/250

which has been released in 2.303.1